### PR TITLE
Ports Floor Cluwnes and Cluwnes.

### PR DESCRIPTION
**Pull request by keronshb**
## About The Pull Request

!KNOHKNOHKNOH ...-tiC ot revo senwulc roolf strop sihT

Cluwnes & the Floor Cluwne midround are added with this PR.

Admins also get to spawn Floor Cluwnes and smite someone into a Cluwne.  Just please remember Cluwning is permanent.  Wizards also get it as a spell.

Some thanks to Putnam and LetterN for helping me with some code issues.

## Why It's Good For The Game

It's nice to have more midround variety, one that's especially terrifying.  Cluwnes feel like they're also part of the Space Station 13 experience so it'll be nice to have them here too.

## Changelog
:cl:
add: Adds Floor Cluwnes and event for midround
add: Adds Cluwne mutation
add: Adds Cluwne spell
add: Adds Cluwnes
imageadd: Adds Cluwne Mask + shoes
admin: Adds Floor Cluwne spawn button
admin: Adds Cluwne smite button
/:cl:

### CODE REVIEW

File: code/datums/mutations/cluwne.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	quality = NEGATIVE
+	locked = TRUE
+	text_gain_indication = "<span class='danger'>You feel like your brain is tearing itself apart.</span>"
+
+/datum/mutation/human/cluwne/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.dna.add_mutation(CLOWNMUT)
+	owner.dna.add_mutation(EPILEPSY)
+	owner.adjustOrganLoss(ORGAN_SLOT_BRAIN, 200, 200)
```
Comment (zeroisthebiggay) [inline code/datums/mutations/cluwne.dm]: doesn't this kill you

**Review by zeroisthebiggay — COMMENTED** [review]
>About The Pull Request

elaborate

>Changelog

he didn't add the cluwne admin smite

**Comment by keronshb** [comment]
> 
> 
> > About The Pull Request
> 
> elaborate
> 
> > Changelog
> 
> he didn't add the cluwne admin smite

I could try to add that smite.

File: code/datums/mutations/cluwne.dm [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+				owner.emote("scream")
+			if(6)
+				owner.Stun(1)
+				owner.Knockdown(20)
+				owner.Jitter(500)
+
+/datum/mutation/human/cluwne/on_losing(mob/living/carbon/human/owner)
+	owner.adjust_fire_stacks(1)
+	owner.IgniteMob()
+	owner.dna.add_mutation(CLUWNEMUT)
```
Comment (SandPoot) [inline code/datums/mutations/cluwne.dm:49]: On losing... you regain it?

Comment (keronshb) [reply code/datums/mutations/cluwne.dm:49]: I think that's the curse of the cluwne, you can never get rid of it.

File: code/datums/mutations/cluwne.dm (hunk 1 above)
Comment (keronshb) [reply code/datums/mutations/cluwne.dm]: Fixed, thank you!

**Pull request merged**